src/parser/test/ParserTest.cpp

```diff
@@ -1694,4 +1694,20 @@ TEST(Parser, Return) {
         ASSERT_TRUE(result.ok()) << result.status();
     }
 }
+
+TEST(Parser, InstallOrUninstallPlugin) {
+    {
+        GQLParser parser;
+        std::string query = "INSTALL PLUGIN authentication SONAME \"authentication_ldap.so\"";
```

Comment:
Could we control the so file's name same with the plugin name？  

if the plugin is named `authentication` and the so is `authentication.so`

Reply:
The plugin  name  and the so file name can be different.
But the so file name must be the same as the function name  we call  in the so file .